assets/tests/components/detours/diversionPageActivate.test.tsx

```diff
@@ -0,0 +1,86 @@
+import React from "react"
```

Comment:
This is a thing I've been thinking about doing for a while.

`diversionPage.test.tsx` is getting very large, which makes it kind of unwieldy. There's no reason we need to have all of the detour tests live in the same file. I was thinking that we could start splitting out test cases for different angles of the detours feature into different test files.

For instance, all of the test cases that make assertions about which route-shape pieces are visible could be in `diversionPageRouteShapes.test.tsx`, and the test cases that assert when the undo and clear buttons are disabled or enabled might live in `diversionPageUndoClearButtons.test.tsx`.

Exactly how to divvy the use cases into files is a matter of taste, and I don't have strong feelings about it. But I figured that test cases related to saving/activating might be happiest away from everything else.